package.json

```diff
@@ -43,13 +43,14 @@
     ]
   },
   "dependencies": {
+    "@1inch/permit-signed-approvals-utils": "^1.4.8",
```

Comment:
😢 1inch utils

Reply:
It does have a lot of stuff abstracted away.
I could re-implement them, but might not be worth the time.